spec/helpers/mock_project.rb

```diff
@@ -109,19 +109,19 @@ def bundle_install(version: nil)
     end
 
     # Run a `command` with `bundle exec` in this project context (unbundled env)
-    sig { params(command: String).returns(ExecResult) }
-    def bundle_exec(command)
+    sig { params(command: String, env: T::Hash[String, String]).returns(ExecResult) }
+    def bundle_exec(command, env = {})
       opts = {}
       opts[:chdir] = path
       Bundler.with_unbundled_env do
-        out, err, status = Open3.capture3(["bundle", "_#{bundler_version}_", "exec", command].join(" "), opts)
+        out, err, status = Open3.capture3(env, ["bundle", "_#{bundler_version}_", "exec", command].join(" "), opts)
         ExecResult.new(out: out, err: err, status: T.must(status.success?))
       end
     end
 
     # Run a Tapioca `command` with `bundle exec` in this project context (unbundled env)
-    sig { params(command: String).returns(ExecResult) }
-    def tapioca(command)
+    sig { params(command: String, enforce_typechecking: T::Boolean).returns(ExecResult) }
+    def tapioca(command, enforce_typechecking = false)
```

Comment:
I'm in favor of always running the tests with this at `true`. Since we have a lot of CLI-tests it will expose more problems we didn't think about when writing unit tests.

Reply:
I've made a separate branch changing the default behaviour for CLI tests (making them only "swallow" errors if you explicitly ask it to): https://github.com/Shopify/tapioca/compare/dont-swallow-errors-on-cli-tests?expand=1 We could treat those `@project.tapioca(<something>, ignore_typechecking_errors: true)` as a "shitlist" while preventing any new CLI tests from implicitly hiding errors. Does that make sense @Morriar @paracycle ?

If so, my plan would be to open a PR with that branch -> merge -> rebase this one (or alternatively I could move everything to this PR, if you prefer).

Reply:
Let's go with another PR for enabling type checking 👍 

Are the elements from the shitlist hard to fix?